components/branding/wordmark/wordmark--products.readme.md

```diff
@@ -5,6 +5,7 @@ Product classes:
 - `mzp-t-product-developer`
 - `mzp-t-product-nightly`
 - `mzp-t-product-focus`
+- `mzp-t-product-klar`
```

Comment:
FYI there's the same list also in `wordmark/readme.md` that could benefit from this addition for consistency, if you think it's worth it.

I'm actually thinking about how to document properly that the Klar component is only among the wordmarks, and trying to use the product class with another branding element will fail and folks have to remember to use Focus classes for all other branding needs — so this is an odd duck, existing only for the text difference here, and not elsewhere where it has no use. Some kind of a note to warn people of how specific it is?